forge-gui/res/cardsfolder/upcoming/hag_of_syphoned_breath.txt

```diff
@@ -0,0 +1,9 @@
+Name:Hag of Syphoned Breath
+ManaCost:2 B
+Types:Creature Hag Warlock
+PT:2/2
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigLoseLife | TriggerDescription$ At the beginning of your upkeep, each opponent loses 2 life and you gain 2 life.
+SVar:TrigLoseLife:DB$ LoseLife | Defined$ Player.Opponent | LifeAmount$ 2 | SubAbility$ DBGainLife
```

Comment:
`Defined$ Opponent`

Also `Defined$ You` is default in line 7 if you want to cut it